hack/ginkgo-e2e.sh

```diff
@@ -41,6 +41,9 @@ GINKGO_NO_COLOR=${GINKGO_NO_COLOR:-n}
 # If 'y', will rerun failed tests once to give them a second chance.
 GINKGO_TOLERATE_FLAKES=${GINKGO_TOLERATE_FLAKES:-n}
 
+# If 'y' the command executed will be `dlv exec` instead of `ginkgo`
+E2E_DEBUG=${E2E_DEBUG:-n}
```

Comment:
I think this should probably reference the tool being used in some fashion, not everyone thinks of this tool when they think of debug 🤔

Reply:
I see, what about `E2E_DEBUG_TOOL=delve`?

Reply:
also, may I ask how do you usually do your e2e debugging?

Reply:
printf :-)
most of the bugs I see involve races
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/flaky-tests.md#deflaking-e2e-tests